### PR TITLE
Fix/PLF-8659:Edit Online button missing in document's preview in search results and in news attachements

### PR DESCRIPTION
**Pull request by ahmadmido**

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js [hunk 1]
```diff
@@ -314,6 +314,9 @@
           '</div>' +
           '<div class="downloadBtn">' +
             '<a href="' + this.settings.doc.downloadUrl + '"><i class="uiIconDownload uiIconWhite"></i>&nbsp;${UIActivity.comment.download}</a>' +
+          '</div>' +
+          '<div class="onlyOfficeEditBtn">' +
+            '<a href="/portal/intranet/oeditor?docId=' + this.settings.doc.id + '" target="_blank"><i class="uiIconEdit uiIconWhite"></i>&nbsp;${UIActivity.label.edit-online}</a>' +
```
Comment (plamarque) [inline ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js]: intranet should not be hardcoded

File: ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js [hunk 2]
```diff
@@ -424,6 +427,9 @@
                     '<div class="downloadBtn">' +
                       '<a href="' + this.settings.doc.downloadUrl + '"><i class="uiIconDownload uiIconWhite"></i>&nbsp;${UIActivity.comment.download}</a>' +
                     '</div>' +
+                    '<div class="onlyOfficeEditBtn">' +
+                      '<a href="/portal/intranet/oeditor?docId=' + this.settings.doc.id + '" target="_blank"><i class="uiIconEdit uiIconWhite"></i>&nbsp;Edit Online</a>' +
+                    '</div>' +
                   '</div>' +
                 '</div>' +
```
Comment (plamarque) [inline ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js]: intranet should not be hardcoded

File: ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js [hunk 3]
```diff
@@ -427,8 +447,26 @@
...
               '</div>';
-
         docPreviewContainer.html(html);
+        let onlyOfficeButton = '';
```
Comment (ahamdi) [inline ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js]: No need to initialize a variable for this, just append it to $(".previewBtn")

File: ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js [hunk 4]
```diff
@@ -427,8 +447,26 @@
...
+                  }
+              }).fail(function(data, textStatus, xhr) {
+                 if( xhr.status === 404)
+                   //OnlyOffice addons is not installed
+                 else {
+                 console.log("Error to call rest service with status code :" +  xhr.status);}
+        });
       }
 
       if(this.settings.doc.remoteEditURL) {
```
Comment (ahamdi) [inline ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js]: Remove the above code and append it to the $(".previewBtn") like it is done here for the remote Edit addon

**Pull request merged**